nginx.conf

```diff
@@ -143,7 +143,7 @@ server {
     # to the URI, resulting in a HTTP 500 error response.
     location ~ \.php(?:$|/) {
         # Required for legacy support
-        rewrite ^/(?!index|remote|public|cron|core\/ajax\/update|status|ocs\/v[12]|updater\/.+|oc[ms]-provider\/.+|.+\/richdocumentscode\/proxy) /index.php$request_uri;
+        rewrite ^/(?!index|remote|public|cron|core\/ajax\/update|status|ocs\/v[12]|updater\/.+|ocs-provider\/.+|.+\/richdocumentscode\/proxy) /index.php$request_uri;
```

Comment:
ocm no longer a thing?

Reply:
This was an attempt to fix collabora - it did not work, and that feature is being punted.  I can revert this?

Reply:
If you don't understand it, it's probably better to leave it as it is